cmd/runtimetest/main.go

```diff
@@ -493,6 +493,25 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	for _, sys := range spec.Linux.Seccomp.Syscalls {
+		if sys.Action == "SCMP_ACT_ERRON" {
+			for _, name := range sys.Names {
+				if name == "getcwd" {
+					_, err := os.Getwd()
+					if err == nil {
+						return fmt.Errorf("when action is %v, return value should being passed to user space as the errno value without executing the system call", sys.Action)
```

Comment:
I don't see [any runtime-oriented RFC 2119 requirements for `linux.seccomp`][1], which means [we only have the config-oriented `SeccSyscallsNamesRequired` in `specerror`][2], which means we have no grounds for making this a fatal error in compliance testing.  I think the best we can do (at least for 1.0.x configs) is warn the caller that the runtime is skipping some not-strictly-required-but-probably-expected functionality.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config-linux.md#seccomp
[2]: https://github.com/opencontainers/runtime-tools/blob/v0.3.0/specerror/config-linux.go#L131

Reply:
Before spec adds RFC requirements, we'd better just give a warn instead of error